product_properties/20-grub.yml

```diff
@@ -0,0 +1,14 @@
+
+{{% if product in ("rhel7", "ol7", "rhel8", "ol8") %}}
+grub2_uefi_boot_path: "/boot/efi/EFI/redhat"
+{{% elif product == "sle12" %}}
+grub2_uefi_boot_path: "/boot/efi/EFI/sles"
+{{% elif "debian" in product %}}
```

Comment:
Why not set these deviations in each product's `product.yml`?
Keeps them local to each product.